release-notes/8.0/known-issues.md

```diff
@@ -2,6 +2,26 @@
 
 You may encounter the following known issues, which may include workarounds, mitigations, or expected resolution timeframes.
 
+## .NET Servicing July 2024 update
```

Comment:
Once we settle on the text above, we'll of course need to replicate it into this file too.

Reply:
ok